tests/o11y/o11y-deployments.go

```diff
@@ -0,0 +1,103 @@
+package o11y
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils"
+)
+
+var _ = framework.O11ySuiteDescribe("O11Y E2E tests for Deployments", Label("o11y", "HACBS"), Pending, func() {
+
+	defer GinkgoRecover()
+	var f *framework.Framework
+	var err error
+
+	Describe("O11y test", func() {
+		var testNamespace string
+
+		BeforeAll(func() {
+
+			f, err = framework.NewFramework(O11yUser)
+			Expect(err).NotTo(HaveOccurred())
+			testNamespace = f.UserNamespace
+
+			// Get Quay Token from ENV
+			quayToken := utils.GetEnv("QUAY_TOKEN", "")
+			Expect(quayToken).ToNot(BeEmpty())
+
+			_, err := f.AsKubeAdmin.CommonController.CreateRegistryAuthSecret(o11yUserSecret, testNamespace, quayToken)
```

Comment:
I think that if both tests (o11y-pipelinerun, o11y-deployment) are run in parallel, they will fail on this, because both tests are using the same namespace name (generated from `O11yUser`)

Reply:
okay will update

Reply:
Done